example/.iex.exs

```diff
@@ -1 +0,0 @@
-import Ecto.Query
```

Comment:
this is to import Ecto.Query functions globally(to Kernel) I dont think its redundant, so I would like to keep it.

Reply:
I forgot to put it back, sorry.